docroot/themes/custom/uids_base/gulpfile.js

```diff
@@ -73,7 +98,7 @@ function watchFiles() {
   // @todo Watch other changes?
 }
 
-const copy = parallel(copyUids, fontCopy);
+const copy = parallel(copyUids, copyColorbox, copyLeaflet, fontCopy);
```

Comment:
Why do libraries need to be copied into uids_base at all?

Reply:
I was just following #1200. I thought they had to be local for use in libraries, etc.

Reply:
IDK off the top of my head. It seems odd that a library would not be usable at the default location. Thats were FontAwesome is located and AFAIK that is working.

Reply:
i can try to use the default paths in the code base and remove if it works.

Reply:
I hate when @richardbporter is right about things like this.

Reply:
Sorry, just curious about things. :)